packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -87,13 +105,18 @@ class MethodWebsocketRequestHandler {
         }
       }
     } catch (e, stackTrace) {
-      if (server.serverpod.runtimeSettings.logMalformedCalls) {
+      if (e is! UnknownMessageException ||
```

Comment:
Discussion: This is the only change that scares me a bit. I'm not really sure in what cases we might have initiated a connection that then fails becuase the client has bad intent.

We might need to expose a server and try this out a bit before including it in the real release.